src/site/xdoc/server/config-elasticsearch.xml

```diff
@@ -38,6 +38,19 @@
           <dd>Is the IP (or host) of the ElasticSearch master</dd>
           <dt><strong>elasticsearch.port</strong></dt>
           <dd>Is the port of ElasticSearch master</dd>
+
+          <dt><strong>elasticsearch.hostScheme</strong></dt>
+          <dd>Only http or https are accepted</dd>
+
+          <dt><strong>elasticsearch.user</strong></dt>
+          <dd>
+              Basic auth username to access elasticsearch.
+              Leave the user and password absent should let James ignore authenticating when connect to elasticsearch.
```

Comment:
when connecting to ES

Or beter: Ignore this property to not be using authentication (default behaviour).